search/routes/api/serialize.py

```diff
@@ -60,63 +65,78 @@ def _transform_license(cls, license: dict) -> dict:
-            'authors': [
+            ]),
+            ('authors', [
                 cls._transform_person(author) for author in doc.authors
                 if author is not None
-            ],
-            'comments': doc.comments,
-            'submitted_date': doc.submitted_date,
-            'submitted_date_first': doc.submitted_date_first,
-            'announced_date_first': (
+            ]),
+            ('comments', doc.comments),
+            ('authors_freeform', doc.authors_freeform),
+            ('submitted_date', doc.submitted_date),
+            ('submitted_date_first', doc.submitted_date_first),
+            ('announced_date_first', (
                 doc.announced_date_first.strftime('%Y-%m')
                 if doc.announced_date_first is not None
                 else None
-            ),
-            'paper_id': doc.paper_id_v,
-            'doi': doc.doi,
-            'formats': [
+            )),
+            ('paper_id', doc.paper_id),
+            ('paper_id_v', doc.paper_id_v),
+            ('doi', doc.doi),
+            ('formats', [
                 cls._transform_format(fmt, doc.paper_id, doc.version)
                 for fmt in doc.formats
-            ],
-            'is_current': doc.is_current,
-            'is_withdrawn': doc.is_withdrawn,
-            'journal_ref': doc.journal_ref,
-            'license': cls._transform_license(doc.license),
-            'msc_class': doc.msc_class,
-            'primary_classification': cls._transform_classification(
-                doc.primary_classification
-            ),
-            'secondary_classification': [
+            ]),
+            ('is_current', doc.is_current),
+            ('is_withdrawn', doc.is_withdrawn),
+            ('journal_ref', doc.journal_ref),
+            ('license',
+             cls._transform_license(doc.license) if doc.license else None),
+            ('msc_class', doc.msc_class),
+            ('primary_classification',
+             cls._transform_classification(doc.primary_classification)
+             if doc.primary_classification else None),
+            ('secondary_classification', [
                 cls._transform_classification(clsn)
                 for clsn in doc.secondary_classification
-            ],
-            'report_num': doc.report_num,
-            'source': doc.source,  # TODO: link?
-            'submitter': (
+            ]),
+            ('report_num', doc.report_num),
+            ('source', doc.source),  # TODO, link?
+            ('submitter', (
                 cls._transform_person(doc.submitter)
                 if doc.submitter is not None else None
-            ),
-            'title': doc.title,
-            'version': doc.version,
-            'latest': cls._transform_latest(doc),
-        }
+            )),
+            ('title', doc.title),
+            ('version', doc.version),
+            ('latest', cls._transform_latest(doc)),
+            ('href', url_for("api.paper", paper_id=doc.paper_id,
```

Comment:
Disregard my previous comment, looks like this might already be taken care of via `_external`